ThinBridge/Chronos
# Drop support for TB_Global_SCHME

# Which issue(s) this PR fixes:

https://github.com/ThinBridge/Chronos/issues/233

# What this PR does / why we need it:

`TB_Global_SCHME` is a typo of `TB_Global_SCHEME`.
We have added support for `TB_Global_SCHEME` and announced `TB_Global_SCHME` is deprecated. Now, we drop support for `TB_Global_SCHME`.

# How to verify the fixed issue:

* Open "設定" -> "リダイレクト設定"
* Check (Enable) "URLリダイレクトを有効にする"
* Open script editor window
* Click the "ヘルプ" button
  * [x] Confirm that there are no description about "TB_Global_SCHME" in the help message
  * [x] Confirm that there are no wrong newlines or lacking of comment outing (`'` of beginning of line)
* Remove help message
* Input the following script
  ```
  Function OnRedirect()
	  TB_TRACE_LOG("TB_Global_URL："&TB_Global_URL)
  	  TB_TRACE_LOG("TB_Global_SCHEME："&TB_Global_SCHEME)
	  TB_TRACE_LOG("TB_Global_HOSTNAME："&TB_Global_HOSTNAME)
	  TB_TRACE_LOG("TB_Global_PORT："&TB_Global_PORT)
	  TB_TRACE_LOG("TB_Global_URL_PATH："&TB_Global_URL_PATH)
  End Function
  ``
* Click the "保存" button
* Close script editor window
* Open "設定" -> "ログ出力設定"
* Open trace log monitor window
* Close the "設定" dialog
* Open some site in the view window
* Open trace log monitor window
* [x] Confirm that logs like below are output
  ```
  11	18:20:14	APP_WND:0x00000000	CSG_Script	TB_Global_URL：https://www.google.co.jp/search
  12	18:20:14	APP_WND:0x00000000	CSG_Script	TB_Global_SCHEME：https
  13	18:20:14	APP_WND:0x00000000	CSG_Script	TB_Global_HOSTNAME：www.google.co.jp
  14	18:20:14	APP_WND:0x00000000	CSG_Script	TB_Global_PORT：443
  15	18:20:14	APP_WND:0x00000000	CSG_Script	TB_Global_URL_PATH：/search
  ```
  * [x] Confirm that `TB_Global_URL` shows URL
  * [x] Confirm that `TB_Global_SCHEME` shows SCHEME
  * [x] Confirm that `TB_Global_HOSTNAME` shows HOSTNAME
  * [x] Confirm that `TB_Global_PORT` shows PORT
  * [x] Confirm that `TB_Global_URL_PATH` shows URL_PATH
